### PR TITLE
chore: improve readme

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ $ make # You can find the binary file in the `_build` dir if all goes good
 ```
 ### Overview
 ![image](docs/images/overview.png)
-For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https://github.com/apache/kvrocks-controller/blob/unstable/storage/persistence/persistence.go).
+For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https://github.com/apache/kvrocks-controller/tree/unstable/store/engine).
```
Comment (caicancai) [inline README.md]: I'm not sure if it's this link, but the previous link is apparently broken

Comment (torwig) [reply README.md]: @caicancai Thank you for the contribution. I think the link can be more precise: https://github.com/apache/kvrocks-controller/blob/unstable/store/engine/engine.go

Comment (caicancai) [reply README.md]: done, thanks

File: README.md [hunk 2]
```diff
@@ -27,12 +27,12 @@ $ make # You can find the binary file in the `_build` dir if all goes good
 ```
 ### Overview
 ![image](docs/images/overview.png)
-For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https://github.com/apache/kvrocks-controller/blob/unstable/storage/persistence/persistence.go).
+For the storage, the ETCD is used as the default storage now. Welcome to contribute other storages like MySQL, Redis, Consul and so on. And what you need to do is to implement the [persistence interface](https ://github.com/apache/kvrocks-controller/blob/unstable/store/engine/engine.go).
```
Comment (torwig) [inline README.md]: Additionally, we can change `persistance` => `Engine`.

Comment (caicancai) [reply README.md]: `persistence interface` => `Engine` 
Maybe it is more appropriate, because Engine is not an interface

Comment (torwig) [reply README.md]: @caicancai Why is `Engine` not an interface?
image

Comment (caicancai) [reply README.md]: @torwig Sorry, I didn't click on the file inside, I thought it was the file name, it's still building. 
Thanks for the correction

Comment (caicancai) [reply README.md]: done

**Review by torwig — APPROVED** [review]
LGTM

**Comment by caicancai** [comment]
@torwig Thank you for your quick response and answer

**Review by git-hulk — APPROVED** [review]

**Pull request merged**